### PR TITLE
feat: add the EIPFS infra for the blob protocol

### PR DESCRIPTION
**Pull request by joaosa**
- adds the EIPFS multihashes queue;
- adds the EIPFS blocks CAR positions table.

### CODE REVIEW

File: .env.tpl [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 EIPFS_INDEXER_SQS_ARN = 'arn:aws:sqs:us-west-2:505595374361:staging-ep-indexer-topic'
 EIPFS_INDEXER_SQS_URL = 'https://sqs.us-west-2.amazonaws.com/505595374361/staging-ep-indexer-topic'
+EIPFS_MULTIHASHES_SQS_ARN = 'arn:aws:sqs:us-west-2:505595374361:multihashes-topic'
```
Comment (alanshaw) [inline .env.tpl:16]: There is no "staging-multihashes-topic"?

File: stacks/upload-api-stack.js [hunk 2]
```diff
@@ -89,6 +116,7 @@ export function UploadApiStack({ stack, app }) {
           PIECE_TABLE_NAME: pieceTable.tableName,
           PIECE_OFFER_QUEUE_URL: pieceOfferQueue.queueUrl,
           FILECOIN_SUBMIT_QUEUE_URL: filecoinSubmitQueue.queueUrl,
+          BLOCKS_CAR_POSITION_TABLE_NAME: blocksCarPositionTable.tableName,
```
Comment (alanshaw) [inline stacks/upload-api-stack.js:119]: Also the multihashes queue URL?

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: .env.tpl (hunk 1 above)
Comment (joaosa) [reply .env.tpl:16]: AFAICT no

File: stacks/upload-api-stack.js (hunk 2 above)
Comment (joaosa) [reply stacks/upload-api-stack.js:119]: alright wasn't sure about that one 👍 . adding it

**Review by alanshaw — APPROVED** [review]
This looks perfect, thanks!

**Review by vasco-santos — APPROVED** [review]
LGTM
Can you please set these env vars in seed.run?

**Comment by joaosa** [comment]
> LGTM Can you please set these env vars in seed.run?

Only missing production. Adding those

**Pull request merged**